src/main.cpp

```diff
@@ -35,16 +35,20 @@ This file is part of DarkStar-server source code.
 
 /* Global Variables */
 xiloader::Language g_Language = xiloader::Language::English; // The language of the loader to be used for polcore.
-std::string g_ServerAddress = "127.0.0.1"; // The server address to connect to.
-std::string g_ServerPort = "51220"; // The server lobby server port to connect to.
-std::string g_LoginDataPort = "54230"; // Login server data port to connect to
-std::string g_LoginViewPort = "54001"; // Login view port to connect to
-std::string g_LoginAuthPort = "54231"; // Login auth port to connect to
-std::string g_Username = ""; // The username being logged in with.
-std::string g_Password = ""; // The password being logged in with.
-char* g_CharacterList = NULL; // Pointer to the character list data being sent from the server.
-bool g_IsRunning = false; // Flag to determine if the network threads should hault.
-bool g_Hide = false; // Determines whether or not to hide the console window after FFXI starts.
+std::string g_ServerAddress   = "127.0.0.1";                 // The server address to connect to.
+std::string g_ServerPort      = "51220";                     // The server lobby server port to connect to.
+std::string g_LoginDataPort   = "54230";                     // Login server data port to connect to
+std::string g_LoginViewPort   = "54001";                     // Login view port to connect to
+std::string g_LoginAuthPort   = "54231";                     // Login auth port to connect to
+std::string g_Username        = "";                          // The username being logged in with.
+std::string g_Password        = "";                          // The password being logged in with.
+char        g_SessionHash[16] = {};                          // Session hash sent from auth
+std::string        g_Email    = "";                           // Email, currently unused
```

Comment:
Dunno if its just my screen, but can you run a clang-format on this?

Reply:
I fixed this in particular, there's a lot of clang-format problems in all the sources and it was pretty nasty. I'll save that for a future PR.